### ISSUE TITLE
[HL2:DM] Missing animation

### ISSUE THREAD

**Issue by awedsa** [ValveSoftware/Source-1-Games#221]
Jumping and switching weapons while in mid-air makes the player model look like this:

![dm_runoff0003](https://f.cloud.github.com/assets/3378854/267827/12e62fc6-8ed0-11e2-8358-964640962d9a.jpg)

System Information:
https://gist.github.com/awedsa/5180221

**Comment by kisak-valve** [comment]
Hello, is anyone still experiencing this issue?

**Comment by Xeogin** [comment]
Yes, this is still an issue @kisak-valve, though a small SourceMod plugin exists with a fix: https://forums.alliedmods.net/showthread.php?p=2404259

**Comment by kisak-valve** [comment]
Thanks for retesting @Xeogin.

**Comment by Combine900** [comment]
2022, Can we please have a fix for that?
